src/labkey/Storage.ts

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2017-2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { buildURL } from './ActionURL';
+import { request } from './Ajax';
+import { getCallbackWrapper, getOnFailure, getOnSuccess, RequestCallbackOptions } from './Utils';
+
+export interface StorageCommandResponse {
+    /** Data object with the key/value pairs from the storage item in the given command */
+    data?: Record<string, any>;
+    /** Success message from the response */
```

Comment:
Only success? Or will it have an error message if `success == false`?

Reply:
Yes, there is only a "message" in the success case. In the error case there is an "exception".